src/extlibs/hrvo/simulated_hrvo_test.cpp

```diff
@@ -199,7 +199,7 @@ TEST_F(SimulatedHRVOTest, test_single_enemy_directly_infront)
 TEST_F(SimulatedHRVOTest, test_zig_zag_movement)
 {
     // The x value of the wall in front of the friendly robot
-    int front_wall_x = -1;
+    int front_wall_x = -2;
```

Comment:
Modified these test values slightly so the destination is not in the enemy defence area anymore.